statusboard/src/components/Map/Map.js

```diff
@@ -13,20 +13,41 @@ import './Map.scss';
 
 export default function Map({ brigadeData, filterOpts, setFilterOpts }) {
   const defaultZoom = 2;
+  const storedCenter = [
+    parseFloat(localStorage.getItem('lat')),
+    parseFloat(localStorage.getItem('lng')),
+  ];
+
   const defaultCenter = [44.967243, -104.771556];
 
-  const [zoom, setZoom] = useState(defaultZoom);
-  const [center, setCenter] = useState(defaultCenter);
+  const [zoom, _setZoom] = useState(
+    localStorage.getItem('zoom') || defaultZoom
+  );
+  const [center, _setCenter] = useState(
+    !Number.isNaN(storedCenter[0]) ? storedCenter : defaultCenter
```

Comment:
Nitpick: Reading a negation is often harder (feel free to ignore this if you don't agree in this case)

```suggestion
    Number.isNaN(storedCenter[0]) ? defaultCenter : storedCenter
```